src/main/java/org/graylog/plugins/outputs/neo4j/Neo4jOutput.java

```diff
@@ -167,7 +129,9 @@ public ConfigurationRequest getRequestedConfiguration() {
                             CK_NEO4J_QUERY, "Cypher query",
                             "MERGE (source:HOST { address: '${source}' })\n" +
                                     "MERGE (user_id:USER { user_id: '${user_id}'})\nMERGE (source)-[:CONNECT]->(user_id)",
-                            "Query will be executed on every log message. Use template substitutions to access message fields: ${took_ms}",
+                            "Query will be executed on every log message.\n" +
+                                    "HTTP Mode: Use template substitutions to access message fields: ${took_ms}\n" +
+                                    "Bolt Mode: Use curly brackets only to access message fields: {took_ms}",
```

Comment:
I would prefer do go with the "curly braces only" syntax because it seems more intuitive for me. Looking in the code I get the point why you went with the other syntax. I think we don't need the JMTE. here. We can send the field mappings as parameters to Neo4j in the REST call (http://neo4j.com/docs/rest-docs/current/#rest-api-use-parameters).